Cargo.toml

```diff
@@ -21,7 +21,7 @@ features = ["extension-module"]
 [dependencies]
 itertools = "0.8"
 failure = "0.1"
-finalfusion = "0.8"
+finalfusion = {git = "https://github.com/finalfusion/finalfusion-rust.git", rev = "da4e8d6"}
```

Comment:
We should postpone merging of this PR until we have a stable `finalfusion` version that has the necessary API changes. We should avoid using git revisions too often.

I will check if we can merge the necessary changes without API breakage (we could then do a minor release).

Reply:
Uploaded `finalfusion` 0.8.2 to crates.io, which adds the necessary traits. So, if you bump the version here to 0.8.2, it should work.